internal/venice-common/src/main/proto/controller/ClusterAdminOpsGrpcService.proto

```diff
@@ -14,6 +14,7 @@ service ClusterAdminOpsGrpcService {
   // AdminTopicMetadata
   rpc getAdminTopicMetadata(AdminTopicMetadataGrpcRequest) returns (AdminTopicMetadataGrpcResponse) {}
   rpc updateAdminTopicMetadata(UpdateAdminTopicMetadataGrpcRequest) returns (UpdateAdminTopicMetadataGrpcResponse) {}
+  rpc updateAdminOperationProtocolVersion(UpdateAdminOperationProtocolVersionGrpcRequest) returns (AdminTopicMetadataGrpcResponse) {}
```

Comment:
I wonder if we should reuse  `UpdateAdminTopicMetadataGrpcRequest` instead of adding new `UpdateAdminOperationProtocolVersionGrpcRequest` type?

Reply:
I was thinking the same! Technically, they should all the same. I was adding new type is because I think it is much clearer to see what variables they can send in. I don't mind reusing the UpdateAdminTopicMetadataGrpcRequest, I will just need extra validation.

Reply:
>. I was adding new type is because I think it is much clearer to see what variables they can send in

I agree. Let's keep it then